ui-tests/src/main/java/io/jenkins/plugins/analysis/warnings/AbstractNonDetailsIssuesTableRow.java

```diff
@@ -0,0 +1,242 @@
+package io.jenkins.plugins.analysis.warnings;
+
+import java.util.List;
+
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+import org.jenkinsci.test.acceptance.po.PageObject;
+
+/**
+ * Abstract representation of a table row displaying an issue.
+ *
+ * @author Stephan Plöderl
+ * @author Anna-Maria Hardi
+ * @author Elvira Hauer
+ */
+public abstract class AbstractNonDetailsIssuesTableRow extends AbstractIssuesTableRow {
+    private static final String SEVERITY = "Severity";
+    private static final String DETAILS = "Details";
+    private static final String AGE = "Age";
+    private static final String FILE = "File";
+    private static final String PACKAGE = "Package";
+    private static final String FILE_LINE_SEPARATOR = ":";
+    private static final By A_TAG = By.tagName("a");
+
+    private final WebElement row;
+    private final IssuesTable issuesTable;
+
+    AbstractNonDetailsIssuesTableRow(final WebElement rowElement, final IssuesTable table) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [It is a good practice to call super() in a constructor](https://app.codacy.com/manual/uhafner/warnings-ng-plugin/pullRequest?prid=5151082)